app/controllers/departments_controller.rb

```diff
@@ -0,0 +1,49 @@
+class DepartmentsController < ApplicationController
+
+  def index
+    @departments = Department.all
+    render json: @departments.to_json, status: 200
+  end
+
+  def create
+    @department = Department.new(department_params)
+
+    if @department.valid?
+      @department.save!
```

Comment:
Why not just do `if @department.save` on line 11?  `save` (as opposed to `save!`) will return false when it fails to save